doc/api/util.md

```diff
@@ -270,6 +270,26 @@ util.formatWithOptions({ colors: true }, 'See object %O', { foo: 42 });
 // when printed to a terminal.
 ```
 
+## util.getNodeReport()
+<!-- YAML
+added: v11.0.0
+-->
+
+* Returns: {string}
+
+Returns the nore report as a string.
+
+Generates a human readable diagnostic summary. The report is intended for
+development, test and production use, to capture and preserve information
+for problem determination. It includes JavaScript and native stack traces,
+heap statistics, platform information and resource usage etc. 
+
+```js
+const util = require('util');
+const report = util.getNodeReport();
```

Comment:
Elsewhere this is `getReport()`

Reply:
The doc has completely changed due to change in the functional interface, and in that process these are fixed.